packages/hydrated_cubit/lib/src/hydrated_storage.dart

```diff
@@ -52,15 +54,20 @@ class HydratedStorage implements Storage {
     return _lock.synchronized(() async {
       if (_instance != null) return _instance;
       final directory = storageDirectory ?? await getTemporaryDirectory();
-      if (!kIsWeb) Hive.init(directory.path);
-      final box = await Hive.openBox<dynamic>(
+      hive = HiveImpl();
```

Comment:
```suggestion
      // Need to use HiveImpl directly to avoid conflicts with existing Hive initialization
      // https://github.com/hivedb/hive/issues/336
      hive = HiveImpl();
```